From 09b9e5a1cc04352dbc1ba59f04f8213a442dd538 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Fri, 5 Jul 2019 16:05:59 -0400
Subject: [PATCH] Remove /MP compiler flag (not compatible with NMake or
 Ninja).

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 635af100d..bf1f4583e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -216,7 +216,6 @@ if(MSVC)
         -D_CRT_NONSTDC_NO_DEPRECATE
     )
     add_definitions(-DHAVE_CONFIG_H)
-    add_definitions(/MP) #build with multiple processors
     add_definitions(/bigobj) #allow for larger object files
 endif(MSVC)
 
-- 
2.20.1

